src/hooks/useToken.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
```

Comment:
do you still need this file?

Reply:
I use it in autorisation

Reply:
oh no, I dont need this

Reply:
should you remove it then?

Reply:
where are you actually making a request to your server?